### ISSUE TITLE
torrent and magnet link pages show blocked by extension message after browser restart

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#7330]
## Description 
Follow up to #6472 

Torrent and magnet link pages show a "blocked by an extension" message after browser restart.

This does not reproduce on C78 versions (1.0.1 Release, 1.2.11 Beta) but does reproduce on C79 versions (1.1.20 Release and 1.2.23 Beta).

## Steps to Reproduce

   1. Clean profile.
   2. Open webtorrent.io/free-torrents.
   3. Click on a torrent or magnet (will encounter #7061 for magnet) link.
   4. Restart Brave.

## Actual result:
Get message below. Refresh of page does not help.
<img width="1195" alt="Screen Shot 2019-12-11 at 11 40 56 AM" src="https://user-images.githubusercontent.com/28145373/70643330-c47f0500-1c0e-11ea-870a-f5dc26434a87.png">
<img width="1192" alt="Screen Shot 2019-12-11 at 11 56 33 AM" src="https://user-images.githubusercontent.com/28145373/70643338-c648c880-1c0e-11ea-8157-3e6ffe9d227c.png">

You can copy/paste the URL into a new tab and that will work.

## Expected result:
Torrent page to be displayed.

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 1.1.20 Chromium: 79.0.3945.74 (Official Build) (64-bit)
-- | --
Revision | 99b6d241cf84f7fa7434ae8a59f9e74eb1464280-refs/branch-heads/3945@{#888}
OS | macOS Version 10.13.6 (Build 17G5019)

Brave | 1.2.23 Chromium: 79.0.3945.70 (Official Build) beta (64-bit)
-- | --
Revision | 65868db27916cebde51fc2465c9b34761e6383e2-refs/branch-heads/3945@{#869}
OS | macOS Version 10.13.6 (Build 17G5019)

## Version/Channel Information:

- Can you reproduce this issue with the current release? yes reproduces with 1.1.20, did not occur with 1.0.1 though
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the dev channel? unsure
- Can you reproduce this issue with the nightly channel? unsure

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? n/a
- Does the issue resolve itself when disabling Brave Rewards? n/a
- Is the issue reproducible on the latest version of Chrome? n/a

**Comment by rebron** [comment]
This is because we're lazy loading the extension, we need to add a check on startup if there's a tab open.

**Comment by feross** [comment]
@bbondy Looks like this case was missed when adding lazy-loading. Is this a fix that is needed for other lazy-loaded extensions too? Can we fix it once for all cases?

**Comment by kjozwiak** [comment]
+1 from https://twitter.com/LuisNil22/status/1281322799634710528 as per @BrendanEich. A quick note, clicking on the refresh button won't work either. User will need to click in the URL and load it again. Example:

![webTorrent](https://user-images.githubusercontent.com/2602313/87094732-669ab600-c20d-11ea-9bbf-be5ffc62eeaa.gif)

**Comment by bbondy** [comment]
Checking if @darkdh or @yrliou might have some cycles to look at this.

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.13.77 Chromium: 85.0.4183.69 (Official Build) dev (64-bit)
-- | --
Revision | 4554ea1a1171bd8d06951a4b7d9336afe6c59967-refs/branch-heads/4183@{#1426}
OS | macOS Version 10.14.6 (Build 18G3020)
```

* Verified STR from description. 
* Confirmed if torrent/magnet link is active tab there is no error shown on the tab on browser restart.
* Confirmed if torrent/magnet link is non-active tab there is no error shown on the tab on browser restart.
* Encountered https://github.com/brave/brave-browser/issues/7061 while testing

------------

**Comment by xlcrr** [comment]
Changing private browsing to normal browsing fixed it for me

**Comment by DylanYoung** [comment]
Is there another issue to fix the private browsing case?  Either it needs a separate setting or it should just be disabled in private browsing and allow regular download.

**Comment by theplaygod97** [comment]
Issue is still present in private browsing tab
